src/zope/security/tests/test_proxy.py

```diff
@@ -1957,7 +1957,11 @@ def test_LocationProxy_gets_a_security_checker_when_importing_z_security(
         import sys
         from zope.location.location import LocationProxy
         import zope.security
-        from zope.security._compat import reload
+        try:
+            from imp import reload
```

Comment:
```
$ python3.6 -Wall -c 'from imp import reload'
-c:1: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
```

We can use `importlib.reload` if we don't care about Python 3.3 and older.

Reply:
Good point, thank you. We dropped 3.3 in #30 so I'll change it.